src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_LENGTH = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int INCREASE_COEFFITIENT = 2;
+    private Node<K, V>[] array;
+    private int size;
+    private int threshold;
+
+    public MyHashMap() {
+        array = new Node[DEFAULT_LENGTH];
+        threshold = (int) (array.length * DEFAULT_LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        increase();
+        putNode(key, value);
     }
 
     @Override
     public V getValue(K key) {
+        Node<K, V> currentNode = array[getIndex(key)];
+        if (currentNode != null) {
+            while (currentNode.next != null
+                    && !(currentNode.key == key || key != null && key.equals(currentNode.key))) {
```

Comment:
1. Change this "currentNode.key == key || key != null && key.equals(currentNode.key))" to Objects.equals(currentNode.key, key). It will be simpliest.

2. IMHO you wil get error if right key is in the last Node.
I think you shoul check thhi after finish loop

Reply:
if the required node is the last one, then its next will be null, the loop will break and its value will be returned